### ISSUE TITLE
Issues with tab  - material-bootstrap-wizard

### ISSUE THREAD

**Issue by kondasMajid** [creativetimofficial/material-bootstrap-wizard#27]
Hi Creative Tim,

Thank you for a great plugin..

While trying to  use your material-bootstrap-wizard  with angular 6, i am facing a problem  with the display..
kindly help me out..

![capture](https://user-images.githubusercontent.com/25379186/44765827-6c9cb780-ab0b-11e8-97b3-64603a7a114a.PNG)

console
![capture](https://user-images.githubusercontent.com/25379186/44766016-4297c500-ab0c-11e8-9ce3-23d21d5a11ff.PNG)

**Comment by dragosct** [comment]
Hi, @kondasMajid! Thank you for using our products. You migrated this product on angular 6 so you are sure that you have the same HTML classes and structure like in our product? 

Regards, 
Dragos

**Comment by kondasMajid** [comment]
Yes @dragosct10 , i did have HTML classes and structure as your product. 
As i closely checked  taking out **material-bootstrap-wizard.js**  from your theme gives the same results as mind, and in my code i have that file imported still not working, probably i'm missing something....
Will be glad for other suggestions..

**Comment by dragosct** [comment]
Hi! It's very possible for the problem to be from the error that you have in your console which tells us that jquery it's not defined and must be fixed that error. 

Regards,
Dragos

**Comment by kondasMajid** [comment]
Yeah surely Jquery ..

**Comment by monikahb** [comment]
Hi @kondasMajid : I am very new to angular 6 and I am getting the same error. Please help me out to resolve this issue in angular 6 .
